### ISSUE TITLE
get error: 'APScheduler' object has no attribute 'task'

### ISSUE THREAD

**Issue by SSCedric** [viniciuschiele/flask-apscheduler#94]
my demo struct
```
.
├── app
│   ├── api_1_0
│   │   ├── apis.py
│   │   ├── __init__.py
│   │   └── __pycache__
│   │       ├── apis.cpython-36.pyc
│   │       ├── __init__.cpython-36.pyc
│   │       └── test.cpython-36.pyc
│   ├── __init__.py
│   ├── models.py
│   ├── __pycache__
│   │   ├── __init__.cpython-36.pyc
│   │   ├── models.cpython-36.pyc
│   │   └── response.cpython-36.pyc
│   ├── response.py
│   ├── static
│   │   ├── css
│   │   ├── fonts
│   │   ├── images
│   │   └── js
│   ├── templates
│   └── utils
│       ├── id_generator.py
│       ├── __init__.py
│       ├── __pycache__
│       │   ├── id_generator.cpython-36.pyc
│       │   └── __init__.cpython-36.pyc
│       └── scheduler_tasks.py
├── config.py
├── dependences.py
├── manager_dev.py
├── manager.py
├── __pycache__
│   └── config.cpython-36.pyc
├── readme.md
├── requirements.txt
└── uwsgi
    └── uwsgi_pro.ini
```

content of config.py

```
import os
from apscheduler.jobstores.sqlalchemy import SQLAlchemyJobStore

class Config(object):
    CACHE_CONFIG = {
            'CACHE_TYPE': 'simple',
            'CACHE_THRESHOLD': 1024,
            'CACHE_DEFAULT_TIMEOUT': 60*5
            }

class DevConfig(Config):
    # 数据库信息从环境变量取
    #SQLALCHEMY_DATABASE_URI = os.environ.get('key')
    SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://demo:demo_pwd@localhost:3306/demo'
    SCHEDULER_API_ENABLED = True
    SCHEDULER_JOBSTORES = {
            'default': SQLAlchemyJobStore(SQLALCHEMY_DATABASE_URI)
            }
    SQLALCHEMY_COMMIT_ON_TEARDOWN = True
    SQLALCHEMY_POOL_SIZE = 50
    SQLALCHEMY_POOL_TIMEOUT = 10
    SQLALCHEMY_POOL_RECYCLE = 60*10
    SQLALCHEMY_MAX_OVERFLOW = 150
    SQLALCHEMY_TRACK_MODIFICATIONS = True
    SQLALCHEMY_ECHO = True
    DEBUG = True

class ProConfig(Config):
    # 数据库信息从环境变量取
    #SQLALCHEMY_DATABASE_URI = os.environ.get('key')
    SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://demo:demo_pwd@localhost:3306/demo'
    SCHEDULER_API_ENABLED = True
    SCHEDULER_JOBSTORES = {
            'default': SQLAlchemyJobStore(SQLALCHEMY_DATABASE_URI)
            }
    SQLALCHEMY_COMMIT_ON_TEARDOWN = True
    SQLALCHEMY_POOL_SIZE = 50
    SQLALCHEMY_POOL_TIMEOUT = 10
    SQLALCHEMY_POOL_RECYCLE = 60*10
    SQLALCHEMY_MAX_OVERFLOW = 50
    DEBUG = False

config = {
            'dev' : DevConfig,
            'pro' : ProConfig,
            'default' : DevConfig
         }
```

content of app/__init__py

```
from flask import Flask
from flask_sqlalchemy import SQLAlchemy
from flask_cache import Cache
from config import config
from flask_apscheduler import APScheduler

db = SQLAlchemy()
cache = Cache()
scheduler = APScheduler()

def create_app(config_name):
    app = Flask(__name__)
    app.config.from_object(config[config_name])

    db.init_app(app)

    cache_config = config[config_name].CACHE_CONFIG
    cache.init_app(app, config=cache_config)

    scheduler.init_app(app)
    scheduler.start()

    from .api_1_0 import api
    app.register_blueprint(api, url_prefix='/api/')

    return app
```

content of app/api_1_0/apis.py

```
from . import api
import requests as req
from ..models import * 
from ..response import Response
from .. import cache
from time import sleep
from datetime import datetime
from .. import scheduler

@scheduler.task("interval", id="do_job_1", seconds=15, misfire_grace_time=900)
def job_1():
    print('{} start task'.format(datetime.now()))
    time.sleep(10)
    print('{} end task'.format(datetime.now()))
```

what i got:
```
(.venv) [root@abc flask_template]# python manager_dev.py runserver
Traceback (most recent call last):
  File "manager_dev.py", line 5, in <module>
    app = create_app('dev')
  File "/prj/dev/flask_template/app/__init__.py", line 23, in create_app
    from .api_1_0 import api
  File "/prj/dev/flask_template/app/api_1_0/__init__.py", line 5, in <module>
    from . import apis
  File "/prj/dev/flask_template/app/api_1_0/apis.py", line 33, in <module>
    @scheduler.task("interval", id="do_job_1", seconds=15, misfire_grace_time=900)
AttributeError: 'APScheduler' object has no attribute 'task'
(.venv) [root@guorou flask_template]# python manager_dev.py runserver
Traceback (most recent call last):
  File "manager_dev.py", line 5, in <module>
    app = create_app('dev')
  File "/prj/dev/flask_template/app/__init__.py", line 23, in create_app
    from .api_1_0 import api
  File "/prj/dev/flask_template/app/api_1_0/__init__.py", line 5, in <module>
    from . import apis
  File "/prj/dev/flask_template/app/api_1_0/apis.py", line 33, in <module>
    @scheduler.task("interval", id="do_job_1", seconds=15, misfire_grace_time=900)
AttributeError: 'APScheduler' object has no attribute 'task'
```

**Comment by CptJason** [comment]
Hello @SSCedric 
Why did you closed this issue? Have it fixed?

**Comment by viniciuschiele** [comment]
The code is in master but it hasn't been released yet, I will do it this week.

**Comment by viniciuschiele** [comment]
I've just released the version 1.11.0
